src/components/ContentHighlights/HighlightStepper/HighlightStepperConfirmContent.jsx

```diff
@@ -104,10 +112,14 @@ const SelectedContent = () => {
       filterString += ')';
     }
     return filterString;
-  }, [currentSelectedRowIds]);
+  }, [currentSelectedRowIds, enterpriseId]);
 
   if (currentSelectedRowIds.length === 0) {
-    return null;
+    return (
+      <Alert data-testid="selected-content-no-results">
```

Comment:
This `Alert` needs a `variant`, otherwise it has a white/transparent background. Try `variant="warning"`